### ISSUE TITLE
Modal - DialogContent warning

### ISSUE THREAD

**Issue by fbeceic** [croz-ltd/tiller#163]
## Basic information

* Tiller version:
  1.7.3

## Bug description

Console (F12) throws Warning:
`Failed prop type: A <DialogContent> must have either an aria-label or aria-labelledby prop.`

when rendering a `Modal` component.

### Steps to reproduce

Code like this:
<img width="430" alt="image" src="https://github.com/croz-ltd/tiller/assets/44879477/615e6b29-f285-427f-89ef-cf64aee57a03">

generates:
<img width="361" alt="image" src="https://github.com/croz-ltd/tiller/assets/44879477/a9aba799-53d6-41ca-a510-aa0b4615c83b">

### Expected behavior

No warning in the console when rendering a `Modal` component.

### Actual behavior

Warning displayed in the console for a `DialogContent` component when rendering a `Modal`.
